package.json

```diff
@@ -12,13 +12,17 @@
     "bundle": "./bundle.sh",
     "postinstall": "NO_DB=true npm run bundle",
     "start": "./run ./src/backend/server.js",
-    "predev": "mustache ./src/frontend/index/development.json ./src/frontend/index/index.mustache > ./src/frontend/public/index.html",
-    "import-cons-group": "babel-node ./scripts/import-cons-group",
-    "dev": "nf start -w --procfile Procfile.dev",
-    "lint": "eslint src",
-    "knex": "knex --knexfile ./src/backend/data/knexfile.js",
-    "migrate": "knex migrate:latest --knexfile ./src/backend/data/knexfile.js",
-    "seed": "knex seed:run --knexfile ./src/backend/data/knexfile.js",
+    "predev": "./node_modules/.bin/mustache ./src/frontend/index/development.json ./src/frontend/index/index.mustache > ./src/frontend/public/index.html",
+    "import-cons-group": "./node_modules/.bin/babel-node ./scripts/import-cons-group",
+    "dev": "./node_modules/.bin/nf start -w --procfile Procfile.dev",
```

Comment:
Also I've got grunt in another branch as well. That's what I've been using to get up and running